homework/index.js

```diff
@@ -30,18 +30,115 @@
     return elem;
   }
 
-  function main(url) {
+  function renderRepositoryDescription(leftContainer, repository) {
+    const parentTable = createAndAppend('table', leftContainer, { id: 'parentTable' });
+    const nameTr = createAndAppend('tr', parentTable, { class: 'descriptions' });
+
+    createAndAppend('td', nameTr, { class: 'titlesName', text: 'Repository:' });
+    createAndAppend('a', nameTr, {
+      class: 'titlesValueRepo',
+      text: repository.name,
+      href: repository.html_url,
+      target: '_blank',
+    });
```

Comment:
You cannot append a `a` element to a `tr`: that produces invalid HTML. A `tr` element can only have `th` and `td` elements as children. Therefore you need to wrap the `a` element inside a `td` element.

This is an example of what you generate now:

```html
<tr class="descriptions">
  <td class="titlesName">Repository:</td>
  <a class="titlesValueRepo" href="https://github.com/HackYourFuture/alumni" target="_blank">
    alumni
  </a>
</tr>
```